### ISSUE TITLE
Collect, label, and pack CBE cables

### ISSUE THREAD

**Issue by rocco8773** [GAPS-Collab/Tasks#60]
We need to collect, label, pack, and ship the PWR and SMA cables for the CBE panels.  The following tables list how many of each cable length are needed for each Panel ID (PID).

### PWR Cables

| PID | Panel type| Panel location | 275 cm | 325 cm | 375 cm |
| --- | --- | --- | --- | --- | --- |
| 01 | 12PP | Top | | 8 | 16 |
| 02 | 12PP | Bot | 12 | 12 | |
| 03 | 08PP | +X | 16 | | |
| 04 | 08PP | +Y | 16 | | |
| 05 | 08PP | -X | 16 | | |
| 06 | 08PP | -Y | 16 | | |
|    | EDGE |    | 8  | | |
| | TOTALS | | 84 | 20 | 16 |

### SMA Cabled

| PID | Panel type| Panel location | 180 cm | 210 cm | 240 cm | 265 cm | 295 cm | 330 cm | 375 cm |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 01 | 12PP | Top |     |     |     |     |     | 24  | 24  |
| 02 | 12PP | Bot |     |     | 8   | 16  | 16  | 8   |     |
| 03 | 08PP | +X  |     | 8   | 12  | 8   | 4   |     |     |
| 04 | 08PP | +Y  | 4   | 12  | 12  | 4   |     |     |     |
| 05 | 08PP | -X  |     | 8   | 12  | 8   | 4   |     |     |
| 06 | 08PP | -Y  | 8   | 12  | 8   | 4   |     |     |     |
|    | EDGE |     |     |     | 16  |     |     |     |     |
|  | TOTALS |     | 12  | 40  | 68  | 40  | 24  | 32  | 24  |

### Front of a RAT

``` bash
|--------------------------------------------------------------------------------------------|
|                                                                                            |
| LTB  (01) (02) (03) (04) (05) (06) (07) (08)     (09) (10) (11) (12) (13) (14) (15) (16)   |
|                                                                                            |
| RB1  (01) (02) (03) (04) (05) (06) (07) (08)                                               |
|                                                                                            |
| RB2  (09) (10) (11) (12) (13) (14) (15) (16)                                               |
|                                                                                            |
| PB       (02)      (04)      (06)      (08)           (10)      (12)      (14)      (16)   |
|     (01)      (03)      (05)      (07)           (09)      (11)      (13)      (15)        |
|                                                                                            |
|--------------------------------------------------------------------------------------------|
```

### Cable numbering

* A label should be placed at both ends of the SMA/PWR cable.
* SMA cables should follow the following pattern:
    * `RAT{RAT ID}{Channel #}-P{Panel ID}-{Paddle ID}{A/B}-{HG/LG}`
    * For example, an SMA connected to the HG channel on side A of paddle 1 (whose in panel 1) and connected to channel 1 of RB1 in RAT 7 would be labeled as `RAT0701-P02-001A-HG`
    * This scheme is organized such that the first group contains the connection info at the RAT and the last three grouping contain the connection info at the paddle.

### CBE Hookup Scheme

![image](https://user-images.githubusercontent.com/29869348/198695061-6dc1bd90-6d4e-4887-8bbf-225a64521691.png)

---

The labeling is now uploaded to the wiki in a spread sheet.  Just look to tabs labeled with `CBE-SMA-HG`, `CBE-SMA-LG`, and `CBE-PWR`.

https://gaps1.astro.ucla.edu/wiki/gaps/images/gaps/0/0e/TOF_sma_pwr_cable_lebeling.xlsx

**Comment by rocco8773** [comment]
@takeruhayashi Just pinging you so this issue is on your radar.

**Comment by rocco8773** [comment]
## Labeling for Panel 01

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 01 | 001 | 08 | 330 | 02 / 01 | `RAT0810-P01-001A-HG` |
| 01 | 002 | 08 | 330 | 04 / 03 |  |
| 01 | 003 | 08 | 330 | 06 / 05 |  |
| 01 | 004 | 08 | 375 | 14 / 13 |  |
| 01 | 005 | 08 | 375 | 16 / 15 |  |
| 01 | 006 | 08 | 375 | 08 / 07 |  |
| 01 | 007 | 16 | 375 | |  |
| 01 | 008 | 16 | 375 | |  |
| 01 | 009 | 16 | 375 | |  |
| 01 | 010 | 16 | 330 | |  |
| 01 | 011 | 16 | 330 | |  |
| 01 | 012 | 16 | 330 | |  |

**Comment by rocco8773** [comment]
# 🚨 DRAFT 🚨

## Labeling for Panel 02

### Readout Board

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 02 | 013 | 10 | 240 | 02 / 01 | `RAT0805-P01-001A-HG` `RAT0806-P01-001B-HG` |
| 02 | 014 | 10 | 265 | 10 / 09 | `RAT0813-P01-002A-HG` `RAT0814-P01-002B-HG` |
| 02 | 015 | 10 | 265 | 04 / 03 | `RAT0803-P01-003A-HG` `RAT0804-P01-003B-HG` |
| 02 | 016 | 10 | 295 | 12 / 11 | `RAT0811-P01-004A-HG` `RAT0812-P01-004B-HG` |
| 02 | 017 | 10 | 295 | 06 / 05 | `RAT0801-P01-005A-HG` `RAT0802-P01-005B-HG` |
| 02 | 018 | 10 | 330 | 14 / 13 | `RAT0809-P01-006A-HG` `RAT0810-P01-006B-HG` |
| 02 | 019 | 14 | 330 | -- / -- | `RAT1616-P01-007A-HG` `RAT1615-P01-007B-HG` |
| 02 | 020 | 14 | 295 | -- / 07 | `RAT1608-P01-008A-HG` `RAT1607-P01-008B-HG` |
| 02 | 021 | 14 | 295 | -- / 13 | `RAT1614-P01-009A-HG` `RAT1613-P01-009B-HG` |
| 02 | 022 | 14 | 265 | -- / 05 | `RAT1606-P01-010A-HG` `RAT1605-P01-010B-HG` |
| 02 | 023 | 14 | 265 | -- / 11 | `RAT1612-P01-011A-HG` `RAT1611-P01-011B-HG` |
| 02 | 024 | 14 | 240 | -- / 03 | `RAT1604-P01-012A-HG` `RAT1603-P01-012B-HG` |

### LTB

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 01 | 001 | 08 | 330 | 11 / 12 | `RAT0811-P01-001A-LG` `RAT0812-P01-001B-LG` |
| 01 | 002 | 08 | 330 | 09 / 10 | `RAT0809-P01-002A-LG` `RAT0810-P01-002B-LG` |
| 01 | 003 | 08 | 330 | 07 / 08 | `RAT0807-P01-003A-LG` `RAT0808-P01-003B-LG` |
| 01 | 004 | 08 | 375 | 05 / 06 | `RAT0805-P01-004A-LG` `RAT0806-P01-004B-LG` |
| 01 | 005 | 08 | 375 | 03 / 04 | `RAT0803-P01-005A-LG` `RAT0804-P01-005B-LG` |
| 01 | 006 | 08 | 375 | 01 / 02 | `RAT0801-P01-006A-LG` `RAT0804-P01-006B-LG` |
| 01 | 007 | 16 | 375 | 16 / 15 | `RAT1616-P01-007A-LG` `RAT1615-P01-007B-LG` |
| 01 | 008 | 16 | 375 | 14 / 13 | `RAT1614-P01-008A-LG` `RAT1613-P01-008B-LG` |
| 01 | 009 | 16 | 375 | 12 / 11 | `RAT1612-P01-009A-LG` `RAT1611-P01-009B-LG` |
| 01 | 010 | 16 | 330 | 10 / 09 | `RAT1610-P01-010A-LG` `RAT1609-P01-010B-LG` |
| 01 | 011 | 16 | 330 | 08 / 07 | `RAT1608-P01-011A-LG` `RAT1607-P01-011B-LG` |
| 01 | 012 | 16 | 330 | 06 / 05 | `RAT1606-P01-012A-LG` `RAT1605-P01-012B-LG` |

### PB

* The PWR cables will hookup to the equivalent channels on the PB as the LG SMA cables connect to on the LTB.
* For the PWR cables, use the same SN as the SMAs but excluded the `-LG` at the end.

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 01 | 001 | 08 | 325 | 11 / 12 | `RAT0811-P01-001A` `RAT0812-P01-001B` |
| 01 | 002 | 08 | 325 | 09 / 10 | `RAT0809-P01-002A` `RAT0810-P01-002B` |
| 01 | 003 | 08 | 375 | 07 / 08 | `RAT0807-P01-003A` `RAT0808-P01-003B` |
| 01 | 004 | 08 | 375 | 05 / 06 | `RAT0805-P01-004A` `RAT0806-P01-004B` |
| 01 | 005 | 08 | 375 | 03 / 04 | `RAT0803-P01-005A` `RAT0804-P01-005B` |
| 01 | 006 | 08 | 375 | 01 / 02 | `RAT0801-P01-006A` `RAT0804-P01-006B` |
| 01 | 007 | 16 | 375 | 16 / 15 | `RAT1616-P01-007A` `RAT1615-P01-007B` |
| 01 | 008 | 16 | 375 | 14 / 13 | `RAT1614-P01-008A` `RAT1613-P01-008B` |
| 01 | 009 | 16 | 375 | 12 / 11 | `RAT1612-P01-009A` `RAT1611-P01-009B` |
| 01 | 010 | 16 | 375 | 10 / 09 | `RAT1610-P01-010A` `RAT1609-P01-010B` |
| 01 | 011 | 16 | 325 | 08 / 07 | `RAT1608-P01-011A` `RAT1607-P01-011B` |
| 01 | 012 | 16 | 325 | 06 / 05 | `RAT1606-P01-012A` `RAT1605-P01-012B` |

**Comment by rocco8773** [comment]
# 🚨 DRAFT 🚨

## Labeling for Panel 03

### Readout Board

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 02 | 013 | 10 | 240 | 02 / 01 | `RAT1002-P02-013A-HG` `RAT1001-P02-013B-HG` |
| 02 | 014 | 10 | 265 | 10 / 09 | `RAT1010-P02-014A-HG` `RAT1009-P02-014B-HG` |
| 02 | 015 | 10 | 265 | 04 / 03 | `RAT1004-P02-015A-HG` `RAT1003-P02-015B-HG` |
| 02 | 016 | 10 | 295 | 12 / 11 | `RAT1012-P02-016A-HG` `RAT1011-P02-016B-HG` |
| 02 | 017 | 10 | 295 | 06 / 05 | `RAT1006-P02-017A-HG` `RAT1005-P02-017B-HG` |
| 02 | 018 | 10 | 330 | 14 / 13 | `RAT1014-P02-018A-HG` `RAT1013-P02-018B-HG` |
| 02 | 019 | 14 | 330 | 11 / 12 | `RAT1411-P02-019A-HG` `RAT1412-P02-019B-HG` |
| 02 | 020 | 14 | 295 | 03 / 04 | `RAT1403-P02-020A-HG` `RAT1404-P02-020B-HG` |
| 02 | 021 | 14 | 295 | 13 / 14 | `RAT1413-P02-021A-HG` `RAT1414-P02-021B-HG` |
| 02 | 022 | 14 | 265 | 05 / 06 | `RAT1405-P02-022A-HG` `RAT1406-P02-022B-HG` |
| 02 | 023 | 14 | 265 | 15 / 16 | `RAT1415-P02-023A-HG` `RAT1416-P02-023B-HG` |
| 02 | 024 | 14 | 240 | 07 / 08 | `RAT1407-P02-024A-HG` `RAT1408-P02-024B-HG` |

### LTB

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 02 | 013 | 10 | 240 | 02 / 01 | `RAT1002-P02-013A-LG` `RAT1001-P02-013B-LG` |
| 02 | 014 | 10 | 265 | 04 / 03 | `RAT1004-P02-014A-LG` `RAT1003-P02-014B-LG` |
| 02 | 015 | 10 | 265 | 06 / 05 | `RAT1006-P02-015A-LG` `RAT1005-P02-015B-LG` |
| 02 | 016 | 10 | 295 | 08 / 07 | `RAT1008-P02-016A-LG` `RAT1007-P02-016B-LG` |
| 02 | 017 | 10 | 295 | 10 / 09 | `RAT1010-P02-017A-LG` `RAT1009-P02-017B-LG` |
| 02 | 018 | 10 | 330 | 12 / 11 | `RAT1012-P02-018A-LG` `RAT1011-P02-018B-LG` |
| 02 | 019 | 14 | 330 | 05 / 06 | `RAT1405-P02-019A-LG` `RAT1406-P02-019B-LG` |
| 02 | 020 | 14 | 295 | 07 / 08 | `RAT1407-P02-020A-LG` `RAT1408-P02-020B-LG` |
| 02 | 021 | 14 | 295 | 09 / 10 | `RAT1409-P02-021A-LG` `RAT1410-P02-021B-LG` |
| 02 | 022 | 14 | 265 | 11 / 12 | `RAT1411-P02-022A-LG` `RAT1412-P02-022B-LG` |
| 02 | 023 | 14 | 265 | 13 / 14 | `RAT1413-P02-023A-LG` `RAT1414-P02-023B-LG` |
| 02 | 024 | 14 | 240 | 15 / 16 | `RAT1415-P02-024A-LG` `RAT1416-P02-024B-LG` |

### PB

* The PWR cables will hookup to the equivalent channels on the PB as the LG SMA cables connect to on the LTB.
* For the PWR cables, use the same SN as the SMAs but excluded the `-LG` at the end.

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 02 | 013 | 10 | 275 | 02 / 01 | `RAT1002-P02-013A` `RAT1001-P02-013B` |
| 02 | 014 | 10 | 275 | 04 / 03 | `RAT1004-P02-014A` `RAT1003-P02-014B` |
| 02 | 015 | 10 | 275 | 06 / 05 | `RAT1006-P02-015A` `RAT1005-P02-015B` |
| 02 | 016 | 10 | 325 | 08 / 07 | `RAT1008-P02-016A` `RAT1007-P02-016B` |
| 02 | 017 | 10 | 325 | 10 / 09 | `RAT1010-P02-017A` `RAT1009-P02-017B` |
| 02 | 018 | 10 | 325 | 12 / 11 | `RAT1012-P02-018A` `RAT1011-P02-018B` |
| 02 | 019 | 14 | 325 | 05 / 06 | `RAT1405-P02-019A` `RAT1406-P02-019B` |
| 02 | 020 | 14 | 325 | 07 / 08 | `RAT1407-P02-020A` `RAT1408-P02-020B` |
| 02 | 021 | 14 | 325 | 09 / 10 | `RAT1409-P02-021A` `RAT1410-P02-021B` |
| 02 | 022 | 14 | 275 | 11 / 12 | `RAT1411-P02-022A` `RAT1412-P02-022B` |
| 02 | 023 | 14 | 275 | 13 / 14 | `RAT1413-P02-023A` `RAT1414-P02-023B` |
| 02 | 024 | 14 | 275 | 15 / 16 | `RAT1415-P02-024A` `RAT1416-P02-024B` |

**Comment by rocco8773** [comment]
# 🚨 DRAFT 🚨

## Labeling for Panel 04

### Readout Board

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 03 | 025 | 08 | 295 | 16 / 15 | `RAT0816-P03-025A-HG` `RAT0815-P03-025B-HG` |
| 03 | 026 | 08 | 265 | 08 / 07 | `RAT0808-P03-026A-HG` `RAT0807-P03-026B-HG` |
| 03 | 027 | 09 | 265 | 14 / 13 | `RAT0914-P03-027A-HG` `RAT0913-P03-027B-HG` |
| 03 | 028 | 09 | 240 | 06 / 05 | `RAT0906-P03-028A-HG` `RAT0905-P03-028B-HG` |
| 03 | 029 | 09 | 240 | 12 / 11 | `RAT0912-P03-029A-HG` `RAT0911-P03-029B-HG` |
| 03 | 030 | 09 | 240 | 04 / 03 | `RAT0904-P03-030A-HG` `RAT0903-P03-030B-HG` |
| 03 | 031 | 09 | 210 | 10 / 09 | `RAT0910-P03-031A-HG` `RAT0909-P03-031B-HG` |
| 03 | 032 | 09 | 210 | 02 / 01 | `RAT0902-P03-032A-HG` `RAT0901-P03-032B-HG` |


### LTB

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 03 | 025 | 08 | 295 | 14 / 13 | `RAT0814-P03-025A-LG` `RAT0813-P03-025B-LG` |
| 03 | 026 | 08 | 265 | 16 / 15 | `RAT0816-P03-026A-LG` `RAT0815-P03-026B-LG` |
| 03 | 027 | 09 | 265 | 12 / 11 | `RAT0912-P03-027A-LG` `RAT0911-P03-027B-LG` |
| 03 | 028 | 09 | 295 | 10 / 09 | `RAT0910-P03-028A-LG` `RAT0909-P03-028B-LG` |
| 03 | 029 | 09 | 240 | 08 / 07 | `RAT0908-P03-029A-LG` `RAT0907-P03-029B-LG` |
| 03 | 030 | 09 | 240 | 06 / 05 | `RAT0906-P03-030A-LG` `RAT0905-P03-030B-LG` |
| 03 | 031 | 09 | 210 | 04 / 03 | `RAT0904-P03-031A-LG` `RAT0903-P03-031B-LG` |
| 03 | 032 | 09 | 210 | 02 / 01 | `RAT0902-P03-032A-LG` `RAT0901-P03-032B-LG` |

### PB

* The PWR cables will hookup to the equivalent channels on the PB as the LG SMA cables connect to on the LTB.
* For the PWR cables, use the same SN as the SMAs but excluded the `-LG` at the end.

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 03 | 025 | 08 | 275 | 14 / 13 | `RAT0814-P03-025A` `RAT0813-P03-025B` |
| 03 | 026 | 08 | 275 | 16 / 15 | `RAT0816-P03-026A` `RAT0815-P03-026B` |
| 03 | 027 | 09 | 275 | 12 / 11 | `RAT0912-P03-027A` `RAT0911-P03-027B` |
| 03 | 028 | 09 | 275 | 10 / 09 | `RAT0910-P03-028A` `RAT0909-P03-028B` |
| 03 | 029 | 09 | 275 | 08 / 07 | `RAT0908-P03-029A` `RAT0907-P03-029B` |
| 03 | 030 | 09 | 275 | 06 / 05 | `RAT0906-P03-030A` `RAT0905-P03-030B` |
| 03 | 031 | 09 | 275 | 04 / 03 | `RAT0904-P03-031A` `RAT0903-P03-031B` |
| 03 | 032 | 09 | 275 | 02 / 01 | `RAT0902-P03-032A` `RAT0901-P03-032B` |

**Comment by rocco8773** [comment]
# 🚨 DRAFT 🚨

## Labeling for Panel 05

### Readout Board

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 04 | 033 | 12 | 265 | 16 / 15 | `RAT1216-P04-033A-HG` `RAT1215-P04-033B-HG` |
| 04 | 034 | 12 | 240 | 08 / 07 | `RAT1208-P04-034A-HG` `RAT1207-P04-034B-HG` |
| 04 | 035 | 12 | 240 | 14 / 13 | `RAT1214-P04-035A-HG` `RAT1213-P04-035B-HG` |
| 04 | 036 | 12 | 240 | 06 / 05 | `RAT1206-P04-036A-HG` `RAT1205-P04-036B-HG` |
| 04 | 037 | 12 | 210 | 12 / 11 | `RAT1212-P04-037A-HG` `RAT1211-P04-037B-HG` |
| 04 | 038 | 12 | 210 | 04 / 03 | `RAT1204-P04-038A-HG` `RAT1203-P04-038B-HG` |
| 04 | 039 | 12 | 210 | 10 / 09 | `RAT1210-P04-039A-HG` `RAT1209-P04-039B-HG` |
| 04 | 040 | 12 | 180 | 02 / 01 | `RAT1202-P04-040A-HG` `RAT1201-P04-040B-HG` |


### LTB

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 04 | 033 | 12 | 265 | 10 / 09 | `RAT1210-P04-033A-LG` `RAT1209-P04-033B-LG` |
| 04 | 034 | 12 | 240 | 08 / 07 | `RAT1208-P04-034A-LG` `RAT1207-P04-034B-LG` |
| 04 | 035 | 12 | 240 | 12 / 11 | `RAT1212-P04-035A-LG` `RAT1211-P04-035B-LG` |
| 04 | 036 | 12 | 240 | 06 / 05 | `RAT1206-P04-036A-LG` `RAT1205-P04-036B-LG` |
| 04 | 037 | 12 | 210 | 14 / 13 | `RAT1214-P04-037A-LG` `RAT1213-P04-037B-LG` |
| 04 | 038 | 12 | 210 | 04 / 03 | `RAT1204-P04-038A-LG` `RAT1203-P04-038B-LG` |
| 04 | 039 | 12 | 210 | 16 / 15 | `RAT1216-P04-039A-LG` `RAT1215-P04-039B-LG` |
| 04 | 040 | 12 | 180 | 02 / 01 | `RAT1202-P04-040A-LG` `RAT1201-P04-040B-LG` |

### PB

* The PWR cables will hookup to the equivalent channels on the PB as the LG SMA cables connect to on the LTB.
* For the PWR cables, use the same SN as the SMAs but excluded the `-LG` at the end.

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 04 | 033 | 12 | 275 | 10 / 09 | `RAT1210-P04-033A` `RAT1209-P04-033B` |
| 04 | 034 | 12 | 275 | 08 / 07 | `RAT1208-P04-034A` `RAT1207-P04-034B` |
| 04 | 035 | 12 | 275 | 12 / 11 | `RAT1212-P04-035A` `RAT1211-P04-035B` |
| 04 | 036 | 12 | 275 | 06 / 05 | `RAT1206-P04-036A` `RAT1205-P04-036B` |
| 04 | 037 | 12 | 275 | 14 / 13 | `RAT1214-P04-037A` `RAT1213-P04-037B` |
| 04 | 038 | 12 | 275 | 04 / 03 | `RAT1204-P04-038A` `RAT1203-P04-038B` |
| 04 | 039 | 12 | 275 | 16 / 15 | `RAT1216-P04-039A` `RAT1215-P04-039B` |
| 04 | 040 | 12 | 275 | 02 / 01 | `RAT1202-P04-040A` `RAT1201-P04-040B` |

**Comment by rocco8773** [comment]
# 🚨 DRAFT 🚨

## Labeling for Panel 06

### Readout Board

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 04 | 033 | 12 | 265 | 16 / 15 | `RAT1216-P04-033A-HG` `RAT1215-P04-033B-HG` |
| 04 | 034 | 12 | 240 | 08 / 07 | `RAT1208-P04-034A-HG` `RAT1207-P04-034B-HG` |
| 04 | 035 | 12 | 240 | 14 / 13 | `RAT1214-P04-035A-HG` `RAT1213-P04-035B-HG` |
| 04 | 036 | 12 | 240 | 06 / 05 | `RAT1206-P04-036A-HG` `RAT1205-P04-036B-HG` |
| 04 | 037 | 12 | 210 | 12 / 11 | `RAT1212-P04-037A-HG` `RAT1211-P04-037B-HG` |
| 04 | 038 | 12 | 210 | 04 / 03 | `RAT1204-P04-038A-HG` `RAT1203-P04-038B-HG` |
| 04 | 039 | 12 | 210 | 10 / 09 | `RAT1210-P04-039A-HG` `RAT1209-P04-039B-HG` |
| 04 | 040 | 12 | 180 | 02 / 01 | `RAT1202-P04-040A-HG` `RAT1201-P04-040B-HG` |


### LTB

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 04 | 033 | 12 | 265 | 10 / 09 | `RAT1210-P04-033A-LG` `RAT1209-P04-033B-LG` |
| 04 | 034 | 12 | 240 | 08 / 07 | `RAT1208-P04-034A-LG` `RAT1207-P04-034B-LG` |
| 04 | 035 | 12 | 240 | 12 / 11 | `RAT1212-P04-035A-LG` `RAT1211-P04-035B-LG` |
| 04 | 036 | 12 | 240 | 06 / 05 | `RAT1206-P04-036A-LG` `RAT1205-P04-036B-LG` |
| 04 | 037 | 12 | 210 | 14 / 13 | `RAT1214-P04-037A-LG` `RAT1213-P04-037B-LG` |
| 04 | 038 | 12 | 210 | 04 / 03 | `RAT1204-P04-038A-LG` `RAT1203-P04-038B-LG` |
| 04 | 039 | 12 | 210 | 16 / 15 | `RAT1216-P04-039A-LG` `RAT1215-P04-039B-LG` |
| 04 | 040 | 12 | 180 | 02 / 01 | `RAT1202-P04-040A-LG` `RAT1201-P04-040B-LG` |

### PB

* The PWR cables will hookup to the equivalent channels on the PB as the LG SMA cables connect to on the LTB.
* For the PWR cables, use the same SN as the SMAs but excluded the `-LG` at the end.

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 04 | 033 | 12 | 275 | 10 / 09 | `RAT1210-P04-033A` `RAT1209-P04-033B` |
| 04 | 034 | 12 | 275 | 08 / 07 | `RAT1208-P04-034A` `RAT1207-P04-034B` |
| 04 | 035 | 12 | 275 | 12 / 11 | `RAT1212-P04-035A` `RAT1211-P04-035B` |
| 04 | 036 | 12 | 275 | 06 / 05 | `RAT1206-P04-036A` `RAT1205-P04-036B` |
| 04 | 037 | 12 | 275 | 14 / 13 | `RAT1214-P04-037A` `RAT1213-P04-037B` |
| 04 | 038 | 12 | 275 | 04 / 03 | `RAT1204-P04-038A` `RAT1203-P04-038B` |
| 04 | 039 | 12 | 275 | 16 / 15 | `RAT1216-P04-039A` `RAT1215-P04-039B` |
| 04 | 040 | 12 | 275 | 02 / 01 | `RAT1202-P04-040A` `RAT1201-P04-040B` |

**Comment by rocco8773** [comment]
# 🚨 DRAFT 🚨

## Labeling for EDGE PADDLES

### Readout Board

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 06 | 049 | 19 | 265 | 16 / 15 | `RAT1916-P06-049A-HG` `RAT1915-P06-049B-HG` |
| 06 | 050 | 19 | 240 | 08 / 07 | `RAT1908-P06-050A-HG` `RAT1907-P06-050B-HG` |
| 06 | 051 | 19 | 240 | 14 / 13 | `RAT1914-P06-051A-HG` `RAT1913-P06-051B-HG` |
| 06 | 052 | 19 | 210 | 06 / 05 | `RAT1906-P06-052A-HG` `RAT1905-P06-052B-HG` |
| 06 | 053 | 19 | 210 | 12 / 11 | `RAT1912-P06-053A-HG` `RAT1911-P06-053B-HG` |
| 06 | 054 | 19 | 210 | 04 / 03 | `RAT1904-P06-054A-HG` `RAT1903-P06-054B-HG` |
| 06 | 055 | 19 | 180 | 10 / 09 | `RAT1910-P06-055A-HG` `RAT1909-P06-055B-HG` |
| 06 | 056 | 19 | 180 | 02 / 01 | `RAT1902-P06-056A-HG` `RAT1901-P06-056B-HG` |


### LTB

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 06 | 049 | 19 | 265 | 10 / 09 | `RAT1910-P06-049A-LG` `RAT1909-P06-049B-LG` |
| 06 | 050 | 19 | 240 | 08 / 07 | `RAT1908-P06-050A-LG` `RAT1907-P06-050B-LG` |
| 06 | 051 | 19 | 240 | 12 / 11 | `RAT1912-P06-051A-LG` `RAT1911-P06-051B-LG` |
| 06 | 052 | 19 | 210 | 06 / 05 | `RAT1906-P06-052A-LG` `RAT1905-P06-052B-LG` |
| 06 | 053 | 19 | 210 | 14 / 13 | `RAT1914-P06-053A-LG` `RAT1913-P06-053B-LG` |
| 06 | 054 | 19 | 210 | 04 / 03 | `RAT1904-P06-054A-LG` `RAT1903-P06-054B-LG` |
| 06 | 055 | 19 | 180 | 16 / 15 | `RAT1916-P06-055A-LG` `RAT1915-P06-055B-LG` |
| 06 | 056 | 19 | 180 | 02 / 01 | `RAT1902-P06-056A-LG` `RAT1901-P06-056B-LG` |

### PB

* The PWR cables will hookup to the equivalent channels on the PB as the LG SMA cables connect to on the LTB.
* For the PWR cables, use the same SN as the SMAs but excluded the `-LG` at the end.

| Panel | Paddle | RAT | Cable | Channel A/B | Serial Num. |
|:---:|:---:|:---:|:---:|:---:| --- |
| 06 | 049 | 19 | 275 | 10 / 09 | `RAT1910-P06-049A` `RAT1909-P06-049B` |
| 06 | 050 | 19 | 275 | 08 / 07 | `RAT1908-P06-050A` `RAT1907-P06-050B` |
| 06 | 051 | 19 | 275 | 12 / 11 | `RAT1912-P06-051A` `RAT1911-P06-051B` |
| 06 | 052 | 19 | 275 | 06 / 05 | `RAT1906-P06-052A` `RAT1905-P06-052B` |
| 06 | 053 | 19 | 275 | 14 / 13 | `RAT1914-P06-053A` `RAT1913-P06-053B` |
| 06 | 054 | 19 | 275 | 04 / 03 | `RAT1904-P06-054A` `RAT1903-P06-054B` |
| 06 | 055 | 19 | 275 | 16 / 15 | `RAT1916-P06-055A` `RAT1915-P06-055B` |
| 06 | 056 | 19 | 275 | 02 / 01 | `RAT1902-P06-056A` `RAT1901-P06-056B` |

**Comment by Jackdickson89** [comment]
Update: Panels 3 and 5 are completely labeled, need to order more shrink wrap labeling

**Comment by takeruhayashi** [comment]
@Jackdickson89 Ordered 4 boxes each (8 cartridges each) on November 4, 2022

**Comment by alkinderm** [comment]
Some of the shrink wrap labels are paired incorrectly beside the printer. This is just a note to double check the pairings before heat-shrinking them to each cable.

**Comment by rocco8773** [comment]
Finished labeling all SMA and PWR cables for panels PID-04 and -06.  The cable bundles are bagged and sitting on the office couch.  The HG SMA cables had a small strip of RED head shrink placed on them.

---

SMA cables for PID-01 are sorted on the table in Vassiliev's lab.  I found a couple miss labeled cables, so I corrected those cables and audited all labeled cables (for PID-01).  The SMA cables are organized on the table with notes specifying what remaining cables need to be labeled for PID-01.
